repositories/pokemon_repo.go

```diff
@@ -0,0 +1,61 @@
+package repositories
+
+import (
+	"encoding/csv"
+	"fmt"
+	"golangBootcamp/m/models"
+	"os"
+	"strconv"
+
+	"github.com/spf13/viper"
+)
+
+type PokemonCSV interface {
+	ReadCsvFile(filePath string) ([][]string, error)
+}
+
+type PokemonRepo struct {
+	pokemonCSV PokemonCSV
+}
+
+func NewPokemonRepo(pokemonCsv PokemonCSV) PokemonRepo {
+	return PokemonRepo{pokemonCsv}
+}
+
+func (pr PokemonRepo) GetPokemonsFromCSV() ([]models.Pokemon, error) {
+	csvPokemons, error := pr.pokemonCSV.ReadCsvFile(viper.GetString("data.pokemon.file"))
```

Comment:
it would be better if you can define your file path as an attrbiute on your repo struct, and then when initializing you could pass the config value on the "constructor" method